templates/download/mediatek-genio/tab-3.html

```diff
@@ -15,6 +15,28 @@ <h2 class="u-hide--small">Genio 350 EVK</h2>
       }}
     </div>
   </div>
+  <div class="row p-strip u-no-padding--top">
+    <hr class="col-9">
```

Comment:
This element should use `p-rule` class name (that would remove the bottom margin from it).

And then the following heading and paragraphs would not need the `u-no-padding--top` on them.